README.md

```diff
@@ -29,3 +29,4 @@ Screenshot: ![](http://i.imgur.com/UAjBySK.png)
 * Death Toll Scans
 * Easy Going Scans
 * Whiteout Scans
+* OneTime Scans
```

Comment:
This should be "One Time Scans!", as that is the same as what they show on their site.